docs/topics/wasm/wasm-overview.md

```diff
@@ -1,61 +1,74 @@
 [//]: # (title: Kotlin Wasm)
 
-> Kotlin Wasm is [Experimental](components-stability.md).
-> It may be changed at any time. Use it only for evaluation purposes.
+> Kotlin Wasm is [Alpha](components-stability.md).
+> It may be changed at any time. You can use it in scenarios before using in production. We would appreciate your feedback on it in [YouTrack](https://kotl.in/issue).
```

Comment:
```suggestion
> It may be changed at any time. You can use it in scenarios before production. We would appreciate your feedback on it in [YouTrack](https://kotl.in/issue).
```